Makefile

```diff
@@ -1,60 +1,24 @@
-.PHONY: install-web
 install-web:
 	cd ./services/web/ && npm install
 
-.PHONY: run-web
 run-web:
 	cd ./services/web/ && npm run dev
 
-.PHONY: install-hooks
-install-hooks:
-	npm install; \
-	npm run prepare; \
-
-.PHONY: install-code-plugins
-install-code-plugins:
-	code --install-extension aaron-bond.better-comments \
-	code --install-extension dbaeumer.vscode-eslint \
-	code --install-extension esbenp.prettier-vscode \
-
-.PHONY: install-gum
 install-gum:
 	go install github.com/charmbracelet/gum@latest
 
-.PHONY: install-air
-install-air:
-	go install github.com/cosmtrek/air@latest
-
 .SILENT: gum
 gum:
 	bash ./cli.sh
 
-.PHONY: install-env
-install-env:
-	cp -n .env.sample .env; \
-	ln -sf ${PWD}/.env ${PWD}/services/web/.env.development; \
-	ln -sf ${PWD}/.env ${PWD}/services/api/.env; \
-	ln -sf ${PWD}/.env ${PWD}/services/url_shortener/.env; \
-	ln -sf ${PWD}/.env ${PWD}/services/py-api/.env; \
-	ln -sf ${PWD}/.env ${PWD}/services/questionnaire;
-
-.PHONY: run-py-api
```

Comment:
Since we are not using this anymore, we should remove it from the postCreateCommand.sh in the .devcontainer folder